### DISCUSSION TITLE
How to add indexes to 'Markets' page?

### DISCUSSION THREAD

**Discussion by dlsnet** [ghostfolio/ghostfolio#1373] [Q&A] [answered]
Hi,

Could you please help me to understand how to add indexes (or any other quotes to that extend) to 'Markets' page? Thank you

<img width="849" alt="image" src="https://user-images.githubusercontent.com/3121405/196230747-07159a24-4125-4529-ad5b-7f0af71d2b94.png">

**Comment by dtslvr** [comment, marked as answer]
Hi @dlsnet 

There is currently no GUI available to configure the benchmarks. But you can set a record directly in the database table `Property`:

```
key: "BENCHMARKS"
value: [{"symbolProfileId":"224c148b-7b97-2d7f-bc94-5fd5cf417034"}]
```

This property is cached, so it can take some minutes until your changes are visible.

**Comment by dlsnet** [comment]
@dtslvr 

It worked, thank you. However, currently it shows change as 'Change from All Time High'. Is it possible to switch it to 'today'?

<img width="771" alt="image" src="https://user-images.githubusercontent.com/3121405/196263852-9c18dffa-15dd-4e62-9708-648d9d21b067.png">
